contracts/test/utils.ts

```diff
@@ -1,7 +1,3 @@
-import { ethers } from "hardhat"
```

Comment:
This file will not compile unless you delete most of it: 
<img width="831" height="217" alt="Screenshot 2025-12-05 at 2 00 52 PM" src="https://github.com/user-attachments/assets/e7edc565-2b6e-4331-848d-a1726149ded2" />